htslib/1.15/Dockerfile

```diff
@@ -0,0 +1,60 @@
+# for easy upgrade later. ARG variables only persist during build time
+ARG HTSLIBVER="1.15"
```

Comment:
I'm being nitpicky here, but I think we can overwrite line 6 with line 2 instead.

So line 6 will be `ARG HTSLIBVER="1.15"` and we delete lines 1-3.

Reply:
Disregard, I should have read the docs: https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact